tests/test_pay.py

```diff
@@ -1425,3 +1426,276 @@ def test_pay_direct(node_factory, bitcoind):
         # have changed.
         l1l2msat = only_one(l1.rpc.getpeer(l2.info['id'])['channels'])['msatoshi_to_us']
         assert l1l2msat == l1l2msatreference
+
```

Comment:
tests for channels in non-`CHANNEL_NORMAL` state?

Reply:
addressed by 06883dd

Reply:
(test_pay `test_setchannelfee_state`)